pandas/tests/extension/base/interface.py

```diff
@@ -35,6 +36,10 @@ def test_array_interface(self, data):
         result = np.array(data)
         assert result[0] == data[0]
 
+        result = np.array(data, dtype=object)
+        expected = np.array(list(data), dtype=object)
+        tm.assert_numpy_array_equal(result, expected)
+
```

Comment:
are we not testing dtype here? its fine, but maybe add an xfail test or a TODO

Reply:
We are testing `dtype` with `dtype=object` ? That's the only dtype for which we know what the result will be, for all other dtypes it depends on the array dtype.